Gemfile

```diff
@@ -87,3 +85,7 @@ group :test do
   gem 'timecop'
   gem 'db-query-matchers'
 end
+
+gem "primer_view_components", "~> 0.1.0"
```

Comment:
Minor thing but can we put these two new gems up before the development and test stuff? Like where the other similar gems are.